Android/app/src/main/java/live/dittolive/chat/data/repository/Repository.kt

```diff
@@ -37,6 +37,10 @@ interface Repository {
 
     fun getDittoSdkVersion(): String
 
+    //authentication
+    fun login()
```

Comment:
Is this part of the authentication branch? Do we want that on main yet?

Reply:
It's not supposed to be on the authentication branch. I see I have login/logout on the UI, though. Let me double check and make a separate PR that does not have authentication, maybe I pulled from the wrong branch

Reply:
This might be the downside of me using a "develop" branch, it's confused things as I think auth got merged to it